Code/GraphMol/MolAlign/Wrap/rdMolAlign.cpp

```diff
@@ -197,6 +208,20 @@ double AlignMolecule(ROMol &prbMol, const ROMol &refMol, int prbCid = -1,
   return rmsd;
 }
 
+double GetBestRMS(ROMol &prbMol, ROMol &refMol, int prbId = -1,
```

Comment:
There's no need to provide default values for the arguments here. The only place this function will be called is from the boost wrapper, and you provide defaults in your `def` call. 
There's nothing wrong with doing it, and other places in the code do this too, but it is redundant.